site/_guides/deploy-options.md

```diff
@@ -13,7 +13,7 @@ If you don't have a cluster with that capability see the [Running without a Kube
 ### Recommended installation details
 
 The recommended installation of Contour is Contour running in a Deployment and Envoy in a Daemonset with TLS securing the gRPC communication between them.
-The [`contour` example]({{site.github.repository_url}}/tree/master/examples/contour/README.md) will install this for you.
+The [`contour` example]({{site.github.repository_url}}/blob/{{ site.github.latest_release.tag_name }}/examples/contour/README.md) will install this for you.
```

Comment:
This resolves to `https://github.com/projectcontour/contour/blob/v0.15.3/examples/contour/README.md` which is 404.